invenio_audit_logs/services/builders.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of Invenio.
+# Copyright (C) 2025 CERN.
+#
+# Invenio-Audit-Logs is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Audit log data building utils."""
+
+from abc import ABC
+
+from flask import current_app
+
+
+class AuditLogBuilder(ABC):
+    """Audit log builder for audit operations."""
+
+    context = None
+
+    action = None
+    resource_type = None
+    message_template = None
+
+    @classmethod
+    def build(cls, resource, action, identity, **kwargs):
+        """Build and register the audit log operation."""
+        if not current_app.config.get("AUDIT_LOGS_ENABLED", False):
+            return
+
+        data = {
+            "action": action,
+            "resource": resource,
+            "user": identity.id,
+            "user_id": str(identity.id),
```

Comment:
this is a bit confusing for me, why `identity.id`is passed to both keys, `user` and `user_id` ?

Reply:
The user key is passed to the entity resolver, where the id is needed